lib/pbench/cli/server/shell.py

```diff
@@ -15,6 +15,7 @@ def app():
     except (ConfigFileNotSpecified, BadConfig) as e:
         print(e)
         sys.exit(1)
+
     return create_app(server_config)
```

Comment:
@npalaska In the process of setting up the development environment of the pbench server, do we manually set `environment` variables or is it handled by something like ansible scripts? (In this case, I  refer to `PBENCH_SERVER_CONFIG`)

Reply:
@aquibbaig I believe we do set it up manually on the server VM

Reply:
`_PBENCH_SERVER_CONFIG` is en environment variable pointing to `/opt/pbench-server/lib/config/pbench-config.cfg` normally. Yes, it has to be manually filled in. The configuration ansible script will parse some of it for setup (like creating the designated directories).

On the agent side there's a `/etc/profile.d/pbench-agent.sh` to set `PYTHONPATH` and `PATH` and config for the binaries; since the server isn't intended for interactive users, the "real" configuration is in the `crontab` file generated in `/opt/pbench-server/lib/crontab/crontab` by the ansible config script. You need to manually review it and then activate it with `crontab crontab` (as `pbench` user) or `crontab -u pbench crontab` as root to complete the server setup. (Without crontab, it'll never process tarballs.)